src/sentry/data/config/sentry.conf.py.default

```diff
@@ -14,6 +14,8 @@ DATABASES = {
         'PASSWORD': '',
         'HOST': '',
         'PORT': '',
+        'AUTOCOMMIT': True,
```

Comment:
Bringing this into the config that `$ sentry init` spits out since we didn't have it there and we should.